railties/lib/rails/tasks/restart.rake

```diff
@@ -1,4 +1,5 @@
 desc "Restart app by touching tmp/restart.txt"
 task :restart do
+  Dir.mkdir('tmp') unless File.directory('tmp')
```

Comment:
If I understand it correctly, `mkdir_p` will work even if the folder is already created:

```
>> require 'fileutils'
=> true
>> Dir.exists?('zomg')
=> false
>> FileUtils.mkdir('zomg')
=> ["zomg"]
>> Dir.exists?('zomg')
=> true
>> FileUtils.mkdir('zomg')
Errno::EEXIST: File exists @ dir_s_mkdir - zomg
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:252:in `mkdir'
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:252:in `fu_mkdir'
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:181:in `block in mkdir'
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:180:in `each'
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/fileutils.rb:180:in `mkdir'
	from (irb):5
	from /Users/godfrey/.rvm/rubies/ruby-2.2.0/bin/irb:11:in `<main>'
>> FileUtils.mkdir_p('zomg')
=> ["zomg"]
>> FileUtils.rmdir('zomg')
=> ["zomg"]
>> Dir.exists?('zomg')
=> false
>> FileUtils.mkdir_p('zomg')
=> ["zomg"]
>> Dir.exists?('zomg')
=> true
```